_exts

```diff
@@ -1 +1 @@
-Subproject commit 52f7bd2216cc22ef52494f346c5643bb2a74513f
+Subproject commit 1b7c4b794d75f7b0eb6749cadccc58f87e3158a0
```

Comment:
we used submodules for travis as I didn't know we could use pip. Let's switch everything to pip :)